### ISSUE TITLE
(python/python3) cup installed pre-release

### ISSUE THREAD

**Issue by lucknaumann** [chocolatey-community/chocolatey-coreteampackages#1513]
Hi. I hope you are well. I had a strange experience with both the "python" and the "python3" packages while updating. I had v3.8.5 from Chocolatey but installed 3.9-a outside of Chocolatey earlier this year.

## Expected Behavior

Chocolatey to **not** upgrade to a pre-release.

## Current Behavior

Chocolatey upgraded to a pre-release.

Here is the output using `cup all`:

```posh
You have python v3.8.5.20200721 installed. Version 3.9.0-b5 is available based on your source(s).
Progress: Downloading python3 3.9.0-b5... 100%
Progress: Downloading python 3.9.0-b5... 100%

python3 v3.9.0-b5 [Approved]
python3 package files upgrade completed. Performing other installation steps.
Installing 64-bit python3...
python3 has been installed.
WARNING: Provided python InstallDir was ignored by the python installer
WARNING: Its probable that you had pre-existing python installation
WARNING: Installed to: C:\Python38
WARNING: Installation folder is not the default. Not changing permissions. Please ensure your installation is secure.
  python3 can be automatically uninstalled.
 The upgrade of python3 was successful.

```

Fortunately, it updated v3.9 (C:\Python39) but updated the Chocolatey packages to v3.9 as well. I have since uninstalled the Chocolatey "python" package but left the "python3" package.

## Steps to Reproduce (for bugs)

1. Have python & python3 v3.8.5 stable installed.
2. Run `cup all` from PS Admin.
3. Wait...
4.

## Context

Well, I don't know if I care that v3.9-b is installed with Chocolatey as long as v3.8.5 has no further updates. I'm not sure I can do a `--side-by-side` install (or want to), especially if the package thinks that v3.9-b is the current release. I will have to use `--version=3.8.5` and not sure if Chocolatey will continue to update that version, provided there are updates. There could be other issues with using `--side-by-side`.

Also, claiming install in the wrong folder is concerning and the comments section on Chocolatey shows this isn't new behavior. Neither are those `WARNING:`s.

## Your Environment

* Package Version used: 3.8.5
* Operating System and version: Windows 10 v1809 (in hopes of upgrading soon...)
* Chocolatey version: Chocolatey v0.10.15
* Install/uninstall gist: I've no idea what this means.

**Comment by AdmiringWorm** [comment]
This is unfortunately not a problem with the package, as the issue you are reporting happens before the package scripts gains control.

You would need to open a issue over at https://github.com/chocolatey/choco (possibly at https://github.com/chocolatey/chocolatey.org) to have this issue looked at.

> Also, claiming install in the wrong folder is concerning and the comments section on Chocolatey shows this isn't new behavior. Neither are those WARNING:s.

That should be have been fixed in the latest stable release of the package, which I pushed last week I believe.
